x-pack/plugins/canvas/public/components/workpad_page/workpad_static_page/workpad_static_page.scss

```diff
@@ -0,0 +1,13 @@
+.canvasStaticPage {
+  z-index: initial;
+  box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.2);
+  position: absolute;
+  top: 0;
+  transform-style: preserve-3d !important;
+  background-color: $euiColorEmptyShade;
+}
+
+.canvasLayout__stage .canvasStaticPage.inactive {
```

Comment:
Following our naming conventions we use verbs for state. so... `.canvasStaticPage.isInactive`

Docs on bottom of this page.

https://elastic.github.io/eui/#/guidelines/sass

Reply:
Done too, thanks for the improvements!